includes/wf_crm_webform_postprocess.inc

```diff
@@ -1513,6 +1513,25 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
         'entity_table' => 'civicrm_contribution',
       );
     }
+    // KG - start
+    // LineItems
+    $fid = "civicrm_1_lineitem_1_contribution_line_total";
+    if (isset($this->enabled[$fid])) {
+      foreach ($this->data['lineitem'][1]['contribution'] as $l => $lineitem) {
+        $fid = "civicrm_1_lineitem_{$l}_contribution_line_total";
+        if ($this->getData($fid) > 0) {
+          $this->line_items[] = array(
+            'qty' => 1,
+            'unit_price' => $lineitem['line_total'],
+            'financial_type_id' => $lineitem['financial_type_id'],
+            'label' => wf_crm_aval($this->node->webform['components'], $this->enabled[$fid] . ':name', t('Line item')),
+            'element' => "civicrm_1_lineitem_{$l}",
```

Comment:
I don't think that's a valid name for a variable.

Reply:
See above - will change that to $n so it is consistent with what we have in the membership section: 

'element' => "civicrm_{$c}_membership_{$n}",

Reply:
Ok - done